CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+- Remove short options -s on hostname_sub, it was already taken
```

Comment:
Hmmm the -s for host_sub was simply not usable... Unless I'm missing something?

Output when adding a puts config in options.rb:
`$ check-graphite-data.rb -s localhost -t test
{:name=>"graphite check", :from=>"-10mins", :until=>"now", :allowed_graphite_age=>60, :server=>"localhost", :target=>"test"}
CheckGraphiteData OK: graphite check value OK
`